### PR TITLE
AG-6207 - Update Chart tick-related documentation.

### PR DESCRIPTION
**Pull request by alantreadway**
https://ag-grid.atlassian.net/browse/AG-6207

Tweak Chart axis tick related documentation after recent tick calculation changes in #5161.

### CODE REVIEW

File: grid-packages/ag-grid-docs/documentation/doc-pages/charts-axes/index.md [hunk 1]
```diff
@@ -66,9 +66,9 @@ represent the same percentage increase. Whereas, if the `number` axis was used,
 The above property of the log axis can also be useful in financial charts. For example, if your rate of
 return on an investment stays consistent over time, the investment value chart will look like a straight line.
 
-By default, the `log` axis renders 10 ticks (and grid lines) per order of magnitude. If your range is wide
-enough, you may start getting too many ticks, in which case using a smaller value for the `tick: { count: xxx }`
-config might be necessary:
+By default, the `log` axis attempts to render 10 ticks (and grid lines) per order of magnitude, depending
```
Comment (manapeirov) [inline grid-packages/ag-grid-docs/documentation/doc-pages/charts-axes/index.md:69]: does it attempt 10?

**Review by manapeirov — APPROVED** [review]
Just one comment to double check as I'm not sure

File: grid-packages/ag-grid-docs/documentation/doc-pages/charts-axes/index.md (hunk 1 above)
Comment (alantreadway) [reply grid-packages/ag-grid-docs/documentation/doc-pages/charts-axes/index.md:69]: From observation yes - as soon as there is sufficient space, there are exponential ticks (with no labels in some cases).

Comment (alantreadway) [reply grid-packages/ag-grid-docs/documentation/doc-pages/charts-axes/index.md:69]: Discussed offline with a log-axis example being resized to sanity-check the statement :-)

**Pull request merged**